server/publications/collections/products.js

```diff
@@ -343,13 +335,17 @@ Meteor.publish("Products", function (productScrollLimit = 24, productFilters, so
       $in: [true, false, null, undefined]
     };
     selector.shopId = {
-      $in: activeShopsIds
+      $in: userAdminShopIds
     };
 
     // Get _ids of top-level products
     const productIds = Products.find(selector, {
       sort,
       limit: productScrollLimit
+    }, {
+      fields: {
+        _id: 1
```

Comment:
not relevant to this PR, but since we only care about the `_id` I figured I'd save some data transfer and request only the `_id` field